requirements.txt

```diff
@@ -8,6 +8,9 @@ torch
 num2words==0.5.10
 word2number==1.1
 
+# for CloseHomophonesSwap
+SoundsLike==0.0.11
+
 # for test cases
```

Comment:
And now to avoid conflicts, you can add this in a new requirements.txt in the folder of "close_homophones_swap"

Reply:
FIxed, thanks for the suggestion.